websrc/gadget_menu/src/gadget_menu_main.tsx

```diff
@@ -730,11 +810,24 @@ class ControlPanel extends React.Component< {}, ControlPanelState >
 			let row = Math.floor( gadgetIndex / 3 );
 
 			let deleteFn = null;
+			let favoriteFn =  null;
```

Comment:
should this be named toggleAutolaunchFn?

Reply:
yep